test/pkg/csi-spec/.gitignore

```diff
@@ -0,0 +1,5 @@
+/protoc
```

Comment:
let's move all the contents of `test/*` to `protosanitizer/test/...`

Reply:
Okay.